code/ship/ship.cpp

```diff
@@ -10046,16 +10046,26 @@ int ship_create(matrix* orient, vec3d* pos, int ship_type, const char* ship_name
 	// but it breaks multi, so there will just be a warning on debug instead.
 	if ((ship_name == nullptr) || (ship_name_lookup(ship_name) >= 0) /*|| (ship_find_exited_ship_by_name(ship_name) >= 0)*/) {
 		// regular name, regular suffix
+		char default_name[NAME_LENGTH];
```

Comment:
I'd call this `base_name` instead; `default_name` refers to the entire thing, including the suffix.

Reply:
Good point, just changed